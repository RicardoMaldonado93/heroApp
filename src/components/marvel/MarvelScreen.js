import React from 'react'

export const MarvelScreen = () => {
    return (
        <div>
            <h1>MARVEL SCREEN</h1>
        </div>
    )
}
