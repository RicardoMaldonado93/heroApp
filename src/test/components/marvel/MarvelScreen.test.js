import {shallow} from 'enzyme';
import React from 'react';
import { MarvelScreen } from '../../../components/marvel/MarvelScreen';

describe('Unitary test for <MarvelScreen />', () => {
    
    const wrapper = shallow( <MarvelScreen /> );

    test('should be match with snapshot', () => {
        expect( wrapper ).toMatchSnapshot();     
    });
    
});
