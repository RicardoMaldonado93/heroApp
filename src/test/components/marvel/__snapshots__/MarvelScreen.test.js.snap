// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unitary test for <MarvelScreen /> should be match with snapshot 1`] = `
<div>
  <h1>
    MARVEL SCREEN
  </h1>
  <hr />
  <HeroesList
    publisher="Marvel Comics"
  />
</div>
`;
